### ISSUE TITLE
All files * placeholder > ENOENT

### ISSUE THREAD

**Issue by franz-josef-kaiser** [jalba/concat-cli#1]
The `*`/placeholder isn't working for me (while a space separated list of files works):

```
concat-cli -f *.js -o concatenated.js
```

throws the following error

```
{ [Error: ENOENT, open 'c:\path\to\files\*.js']
  errno: 34,
  code: 'ENOENT',
  path: 'c:\\path\\to\\files\\*.js' }
```

CLI: Git/GNU Bash, Platform/OS Win 7

**Comment by jalba** [comment]
It looks like it's not finding any js files in the directory where you are calling the concat-cli module.

**Comment by franz-josef-kaiser** [comment]
It looks like that, but in reality there are files and it's working when I simply use a space separated list of files. Example:

```
concat-cli -f one.js two.js -o concatenated.js
```

This is _ok_ as long as there are only two or three files. As soon as there are more (or longer file names) it gets a nightmare.

**Comment by mikerockett** [comment]
I'm experiencing the same issue. Using `npm` as a task runner / build tool. The task I've defined is as follows:

``` json
"concat:ecma": "concat-cli -f src/ecma/*.js -o src/ecma/build.js",
```

Running `npm run concat:ecma` throws the below-mentioned error, whilst running the command manually works fine.

```
{ [Error: ENOENT, open '<path>\assets.package\src\ecma\*.js']
  errno: -4058,
  code: 'ENOENT',
  path: '<path>\\assets.package\\src\\ecma\\*.js' }
```

Note that I am using Windows - doubt it would ake a difference though as I'm using `mingw32`.

**Comment by mikerockett** [comment]
Right, the issue is my use of `mingw32`. When you call `npm` there, it doesn't pipe instructions to `mingw32`, which means that globs are not available.

Right now, `npm` does not seem to have support for `mingw32`.

Better to use Cygwin, though I have not tried as yet.

**Comment by dmitrijs** [comment]
+1

**Comment by simeydotme** [comment]
I just came across this issue with NPM Scripts... happens on Console2 and Babun

**Comment by franz-josef-kaiser** [comment]
@jalba Are there enough people who have confirmed this to fix it?

**Comment by MNBuyskih** [comment]
Fix this bug and extends functionality by using glob npm package.

Please approve my pull request.

**Comment by denis1stomin** [comment]
Still does not work on Windows 10. Running from cmd or inside npm task.
concat-cli -f _somedir/**/*.js -o _temp/output.js
